CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on format provided by [Keep a Changelog](https://keepachangelog.com) website.
+
+## [Unreleased]
+
```

Comment:
Add some more information about upcoming features for the next release.

Reply:
Added.